docs/circuit_diagrams/module_PI/README.md

```diff
@@ -0,0 +1,40 @@
+The Raspberry Pi Module
+===============
+
+The **Raspberry PI module** integrates a [Raspberry Pi 3](https://www.raspberrypi.org/products/raspberry-pi-3-model-b/) into the circuit to add another layer of functionality. It is initially intended to include the following features:
+
+*A 7'' Touchscreen to allow for a GUI
+*A timestamped logging system
+*A way to manage users and keycodes
+*Email/Text Message alerts
+*SSH control
+*USB webcam support
+*A web interface
+
+
+
+Parts
+-----
+* [Raspberry Pi 3 Model B](https://www.raspberrypi.org/products/raspberry-pi-3-model-b/)  
+  -[This kit](https://www.amazon.com/LoveRPi-Raspberry-Plug-Play-Starter/dp/B01IYBZEV6/ref=sr_1_16?s=pc&ie=UTF8&qid=1491114514&sr=1-16&keywords=raspberry+pi+3) comes with all the hardware you need to get started.
+* [7 Inch Touchscreen](https://www.amazon.com/Raspberry-Pi-7-Touchscreen-Display/dp/B0153R2A9I/ref=sr_1_1?s=electronics&ie=UTF8&qid=1491114627&sr=1-1&keywords=7+inch+touchscreen+raspberry+pi)
+* [Raspberry Pi Touchscreen Case](https://www.amazon.com/Case-Official-Raspberry-Touchscreen-Display/dp/B01HV97F64/ref=pd_sim_147_1?_encoding=UTF8&psc=1&refRID=HCS99YG62WD3F73SRZXA)
+
+
+Setup
```

Comment:
I know that it is as simple as plugging in the USB cable from the Pi to the Arduino to have them communicate, but lets state that somewhere in this section. Or, to be more consistent with our other diagrams, you could add a Wiring section and just make a statement about just hooking the usb ports together. Someone who doesn't know how you've implemented it might be wondering how to get that to work.

There also should be some sort of diagram or statement about how the touchscreen is wired up as well. I don't have a clue what that wires up like to the Pi so I'm as in the dark as someone external would be.